README.md

```diff
@@ -1 +1,42 @@
-# geckos-14
\ No newline at end of file
+# DashThings.io
+
+DashThing.io is tool dedicated to DIYers in need of an easy way to build
```

Comment:
There seems to be an inconsistency of our app in this file

Reply:
Ok will fix it :)
Add 'a'